src/job/models/ingestion/seedingJobCreator.ts

```diff
@@ -98,14 +97,87 @@ export class SeedingJobCreator {
     });
   }
 
-  private createSeedOptions(mode: SeedMode, geometry: Footprint, cacheName: string): SeedTaskOptions {
+  private handleCleanMode(
+    job: IngestionUpdateFinalizeJob | IngestionSwapUpdateFinalizeJob,
+    cacheName: string,
+    catalogId: string
+  ): ICreateTaskBody<SeedTaskParams> | void {
+    const activeSpan = trace.getActiveSpan();
+    const seedTaskType = this.tilesSeedingConfig.type;
+    const logger = this.logger.child({ mode: SeedMode.CLEAN });
+
+    const cleanGeometry = this.calculateGeometryByMode(SeedMode.CLEAN, job);
```

Comment:
"normal" update ingestion would have to clean only by the unify parts (same as the seed footprint) just in "clean" mode and not the entire layer footprint as it now